src/README.md

```diff
@@ -239,5 +239,29 @@ juju config trove nova-keypair=trove-keypair
 ```
 
 
+## Troubleshooting
+
+Each database instance deployed has **2 ports** for the:
+- `tenant-net`
+- `trove-net`
+
+The **tenant-net port** has a security group (`trove_sg-uuid`) that allows the RabbitMQ traffic.
+
+The **trove-net port** has a security group (`trove-sec-group`) that allows the egress traffic 
+through the `mgmt-net`.
+
+**NOTE**: A `trove_sg-uuid` security group is created for every instance.
+**NOTE**: The `trove-sec-group` security group is shared across the instances.
+
+In order to access the database instances through the public IP, a `security group rule` 
+that enables access must be added to the `trove_sg-uuid` to allow connection.
+
+```bash
+# An example on how to add a ssh rule to your instance
+openstack security group rule create trove_sg-uuid \
+--protocol tcp --dst-port 22:23 --ethertype ipv4
```

Comment:
should indent this line by 2 spaces, as it is typical for multi line commands.

Reply:
Done